packages/core-api/lib/versions/1/transformers/delegate.js

```diff
@@ -12,7 +12,7 @@ module.exports = (delegate) => {
     username: delegate.username,
     address: delegate.address,
     publicKey: delegate.publicKey,
-    votes: delegate.votebalance,
+    vote: delegate.votebalance + '',
```

Comment:
Something to keep in mind when converting to string. If for some reason the `votebalance` is `1E-8`, then `votebalance + ''` will still return `number`.

Reply:
Can you elaborate on this? `typeof(0.00000001 + '')` returns `String` for me, same goes for `typeof(1E-8 + '')` or am I misunderstanding you?

Reply:
Nevermind, ignore my comment.